amivapi/cron.py

```diff
@@ -99,7 +99,11 @@ def wrapped():
 
 def schedule_task(time, func, *args):
     """ Schedule a task at some point in the future. """
-    func_s = func_str(func)
+    if func_str(func) is "remindermail":
+        item = pickle.load(args)
+        func_s = "remindermail"+str(item['_id'])
+    else:
+        func_s = func_str(func)
```

Comment:
This seems super hacky to me. Is this special case for reminder mails really necessary?